tests/unit/test_db_rds.py

```diff
@@ -94,5 +94,7 @@ def test_rds_testfree_postgres_urlencoded(self):
         )
         assert config["DatabaseName"] == "dbuajsdhkasdhaks"
         assert config["DatabaseJdbcUrl"].find("tcpKeepAlive") >= 0
+        assert config["DatabaseJdbcUrl"].find("sslmode") >= 0
+        assert config["DatabaseJdbcUrl"].find("sslfactory") >= 0
```

Comment:
Why those parameters may not be in the config?

Reply:
These parameters will be in the config only when database hostname matches `*.rds.amazonaws.com` pattern (which is the case in `test_db_rds.py` testcase). So, for instance, with `config["DatabaseHost"] == "10.10.1.100:5432"` in database configuration the `sslmode` and `sslfactory` parameters are not expected in JDBC URL.